plugins/ui/make_icon_types.py

```diff
@@ -1,80 +1,72 @@
 import re
+from typing import Dict, Any
 
 
 def camel_to_snake(name: str) -> str:
     s1 = re.sub("(.)([A-Z][a-z]+)", r"\1_\2", name)
     return re.sub("([a-z0-9])([A-Z])", r"\1_\2", s1).lower()
 
 
+def update_dict(
+    dictionary: Dict[str, str], key: str, value: str, overwrite_condition: bool
+) -> None:
+    if key in dictionary:
+        if overwrite_condition:
+            dictionary[key] = value
+    else:
+        dictionary[key] = value
```

Comment:
Can combine this into one if statement
```suggestion
    if overwrite_condition or not key in dictionary:
        dictionary[key] = value
```